test-lib.sh

```diff
@@ -867,7 +867,7 @@ ct_test_app_dockerfile() {
 
   # If app_url contains @, the string after @ is considered
   # as a name of a branch to clone instead of the main/master branch
-  readarray -d "@" -t git_url_parts<<<"${app_url}"
+  IFS='@' read -ra git_url_parts <<< "${app_url}"
```

Comment:
I would be glad to have here `old_IFS` and after this row returns back.

Like here and bellow https://github.com/sclorg/container-common-scripts/blob/master/test-lib.sh#L78
Take an inspiration.

Reply:
And here returns back https://github.com/sclorg/container-common-scripts/blob/master/test-lib.sh#L91

Reply:
I don't think it's necessary to switch the variable back if you change it for only one command. Tested on Centos 7:
```
# echo $IFS

# IFS='@' read -ra git_url_parts <<< "test@testbranch"
# echo ${git_url_parts[0]}
test
# echo ${git_url_parts[1]}
testbranch
# echo $IFS

#
```

In this way, the IFS is an environment variable passed to one command only instead of a shell variable modified for subsequent commands.

Reply:
@phracek I think the use case you linked is different, because you set IFS for the entire environment, but Lumir is setting it only for that one command, so it's not necessary to restore the variable afterwards.